src/dotnetTestExplorer.ts

```diff
@@ -177,6 +178,22 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
 
     /**
      * @description
+     * Gets the options for build/restore before running tests.
+     */
+    private getDotNetTestOptions(): string {
+        return this.checkBuildOption() + this.checkRestoreOption();
+    }
+
+    /**
+     * @description
+     * Gets the dotnet test argument to speicfy the output for the test results.
+     */
+    private outputTestResults(): string {
+        return " --logger 'trx;LogFileName=" + this.resultsFile.fileName + "'";

```

Comment:
I test it on Windows, it is failed, After I change single quote to double quote, it is working. How about you?

Reply:
![image](https://user-images.githubusercontent.com/1050213/32825106-9f15124e-ca1e-11e7-8155-b98cebd2e3d2.png)

Reply:
It did work for me, but I've replaced with double quotes and it still works for me so hopefully that should fix it for you

Reply:
I guess you are using PowerShell. Both are working on PowerShell, while single quote does not work on CMD.